shared_bills/urls.py

```diff
@@ -7,7 +7,12 @@
 from rest_framework.routers import APIRootView
 
 
-api_root_views = {"events": "events-list", "user": "user-detail"}
+api_root_views = {
+    "events": "events-list",
+    "user": "user-detail",
+    "get token": "token_obtain_pair",
+    "verify token": "token_verify",
```

Comment:
Urls are not visible in api root view. Use `-` instead of `_` as in the view names set in ` accounts/urls.py `

Reply:
Fixed